modules/pytket-quantinuum/pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -371,6 +376,8 @@ def process_circuits(
         group = kwargs.get("group")
```

Comment:
```suggestion
        group = kwargs.get("group", self._group)
```

then the new elif block can be removed

Reply:
nice!